protocol/x/clob/keeper/authenticators.go

```diff
@@ -0,0 +1,73 @@
+package keeper
```

Comment:
For my understanding, this PR doesn't need to touch `ProcessOperations` explicitly, because it's already taken care of by calling `AnteHandler` on the orders right?

Reply:
correct. 

again, the assumption here is that the light weight validation is sufficient because all current validations are stateless

Reply:
Correct, and `ProcessOperations` do run the anthenticators explicitly since we cannot trust the proposer